services/wiki/wiki.go

```diff
@@ -366,3 +366,13 @@ func DeleteWikiPage(doer *models.User, repo *models.Repository, wikiName string)
 
 	return nil
 }
+
+// DeleteWiki removes the actual and local copy of repository wiki.
+func DeleteWiki(repo *models.Repository) error {
+	if err := repo.DeleteWiki(); err != nil {
```

Comment:
use `models.UpdateRepositoryUnits(repo, nil, []models.UnitType{UnitTypeWiki})` to be able to rm repo.DeleteWiki() func